versions.yaml

```diff
@@ -78,6 +90,27 @@ versions:
       redis:
         sha256: 6624841267e142c5d5d5be292d705f8fb6070677687c5aad1645421a936d22b3
         version: '5.0.6'
+    excludeTests:
+    - tests/functional_tests/redis_exporter_test.yaml
 
+
+  # Redis Exporter for Prometheus
+  - dir: 'exporter'
+    templateSubDir: exporter
+    repo: redis5
+    tags:
+      - 'exporter'
+      - 'exporter-1.3'
+      - 'exporter-1.3.1'
+    from: *from9
+    packages:
+      exporter:
+        version: '1.3.1'
+        sha1: d4459037d64b07a6ca7d316904e246b729930a69822301d0c7a018913c9c1fd1
```

Comment:
This is also not used, right?

Please remove.

Reply:
Fixed.